Makefile

```diff
@@ -0,0 +1,42 @@
+PROG_NAME=olm-mermaid-graph
+#OUTPUT_TYPE=png # or pdf
+OUTPUT_TYPE=svg
+MERMAID_TEMP_SCRIPT=mermaid.mer
+#INDEX_DB_PATH_AND_NAME?=/Users/btofel/workspace/sample-operator/test-registry.db
+#INDEX_DB_PATH_AND_NAME?=olm_catalog_indexes/index.db.4.6.community-operators
+INDEX_DB_PATH_AND_NAME?=olm_catalog_indexes/index.db.4.6.redhat-operators
+
+detected_OS := $(shell uname 2>/dev/null || echo Unknown)
+
+.PHONY: all
+all: build
+
+.PHONY: build
+build:
+	go build -o bin/$(PROG_NAME) ./
+
+.PHONY: install
+install:
+	go install
+
+run: build
+	sed 's+olm_catalog_indexes/index.db.4.6.redhat-operators+$(INDEX_DB_PATH_AND_NAME)+' sqlite3.sql > sqlite3_exec.sql
+	sqlite3 -bail -init sqlite3_exec.sql 2>/dev/null | bin/$(PROG_NAME) $(ARGS) 1>$(MERMAID_TEMP_SCRIPT)
+	docker pull minlag/mermaid-cli
+	docker run \
+		-v $(PWD)/$(MERMAID_TEMP_SCRIPT):/$(MERMAID_TEMP_SCRIPT) \
+		-v $(PWD)/config.json:/config.json \
+		-v /tmp:/tmp -it \
+		docker.io/minlag/mermaid-cli:latest \
+		-c /config.json -i /$(MERMAID_TEMP_SCRIPT) -o /tmp/$(MERMAID_TEMP_SCRIPT).$(OUTPUT_TYPE)
+	echo "output $(OUTPUT_TYPE) file is /tmp/$(MERMAID_TEMP_SCRIPT).$(OUTPUT_TYPE)"
+ifeq ($(detected_OS),Darwin)
+	open /tmp/$(MERMAID_TEMP_SCRIPT).$(OUTPUT_TYPE)
+endif
+
+.PHONY: clean
+clean:
+	rm -r bin
+	rm mermaid.mer
+	rm /tmp/mermaid.mer.png
+
```

Comment:
could we add make generate-sample that runs the report and add in the testdata/ dir?

Reply:
I think what's wanted for this `make` target is to just do a sample run, right? But what operator is sure to be there, not change name, etc? Would prefer to leave with just the mention of typical use in the `README.txt` than create a potentially non-working target.

Reply:
Why we need a make target to run? The project is using cobra so we ought to work with flags. E.g bin --image --pakage

The generate-sample suggestion made here is for we have a testdata with samples report. See that is common the tools have samples of what its output. Then, testdata has a special meaning for Golang. Anything that is not part of the code such as mocks or samples goes under that.